flink-runtime/src/main/java/org/apache/flink/runtime/io/network/partition/consumer/InputChannel.java

```diff
@@ -311,6 +311,12 @@ public long unsynchronizedGetSizeOfQueuedBuffers() {
         return 0;
     }
 
+    // ------------------------------------------------------------------------
+    //  For tiered storage
+    // ------------------------------------------------------------------------
+
+    public void notifyRequiredSegmentId(int segmentId) {}
```

Comment:
What is the semantic here? Who should implement it? If this will be completed in future, we should add a `TODO` mark. Or if this is meant to be overridden, we should explain it in comment.